ios/Video/NewPlayerView.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  RNNewPlayerView.swift
+//  RNDReactNativeDiceVideo
+//
+//  Created by Yaroslav Lvov on 05.03.2021.
+//
+
+import React
+import AVDoris
+import AVKit
+import RNDReactNativeDiceVideo
+
+class NewPlayerView: UIView, JSInputProtocol {
+    var jsBridge: RCTBridge?
+    //Events
+    @objc var onBackButton: RCTBubblingEventBlock?
+    @objc var onVideoLoadStart: RCTBubblingEventBlock?
+    @objc var onVideoLoad: RCTBubblingEventBlock?
+    @objc var onVideoBuffer: RCTBubblingEventBlock?
+    @objc var onVideoError: RCTBubblingEventBlock?
+    @objc var onVideoProgress: RCTBubblingEventBlock?
+    @objc var onVideoSeek: RCTBubblingEventBlock?
+    @objc var onVideoEnd: RCTBubblingEventBlock?
+    @objc var onTimedMetadata: RCTBubblingEventBlock?
+    @objc var onVideoAudioBecomingNoisy: RCTBubblingEventBlock?
+    @objc var onVideoFullscreenPlayerWillPresent: RCTBubblingEventBlock?
+    @objc var onVideoFullscreenPlayerDidPresent: RCTBubblingEventBlock?
+    @objc var onVideoFullscreenPlayerWillDismiss: RCTBubblingEventBlock?
+    @objc var onVideoFullscreenPlayerDidDismiss: RCTBubblingEventBlock?
+    @objc var onReadyForDisplay: RCTBubblingEventBlock?
+    @objc var onPlaybackStalled: RCTBubblingEventBlock?
+    @objc var onPlaybackResume: RCTBubblingEventBlock?
+    @objc var onPlaybackRateChange: RCTBubblingEventBlock?
+    @objc var onRequireAdParameters: RCTBubblingEventBlock?
+    @objc var onVideoAboutToEnd: RCTBubblingEventBlock?
+    @objc var onFavouriteButtonClick: RCTBubblingEventBlock?
+    @objc var onRelatedVideoClicked: RCTBubblingEventBlock?
+    @objc var onRelatedVideosIconClicked: RCTBubblingEventBlock?
+    @objc var onStatsIconClick: RCTBubblingEventBlock?
+    @objc var onEpgIconClick: RCTBubblingEventBlock?
+    @objc var onAnnotationsButtonClick: RCTBubblingEventBlock?
+    @objc var onSubtitleTrackChanged: RCTBubblingEventBlock?
+    
```

Comment:
can you please check old implementation where these callbacks are used and split them here with comment

//used
//not used

we can not just ignore callbacks if they are missing in rndv, if they are missing need to add those in rndv